editor/selectors.js

```diff
@@ -168,19 +168,39 @@ export function getEditedPostVisibility( state ) {
 }
 
 /**
- * Return true if the post being edited has already been published.
+ * Return true if the current post has already been published.
  *
  * @param  {Object}   state Global application state
  * @return {Boolearn}       Whether the post has been bublished
  */
-export function isEditedPostPublished( state ) {
+export function isCurrentPostPublished( state ) {
 	const post = getCurrentPost( state );
 
 	return [ 'publish', 'private' ].indexOf( post.status ) !== -1 ||
 		( post.status === 'future' && moment( post.date ).isBefore( moment() ) );
 }
 
 /**
+ * Return true if the post being edited has already been published.
+ *
+ * @param  {Object}   state Global application state
+ * @return {Boolearn}       Whether the post has been bublished
+ */
+export function isEditedPostPublished( state ) {
```

Comment:
I'm having trouble understanding if this is really necessary because the status is changed to 'publish', 'private' only when saving. so no real difference with `isCurrentPostPublished`. Btw, good renaming.